src/scss/components/form/_index.scss

```diff
@@ -151,5 +151,26 @@ $form-asterisk-offset: $form-asterisk-size + $form-asterisk-gap;
   }
 }
 
+.align {
+  display: inline;
+}
+
+hx-text-control {
+  .hxErrorText {
+    color: $red-900;
+    font-size: 0.75rem;
+    margin-left: inherit;
+    line-height: 1.5;
+    display: block;
+  }
+
+  .hxHelpText {
+    margin-left: inherit;
+    display: block;
+    font-size: 0.75rem;
+    line-height: 1.5;
+  }
+}
+
```

Comment:
These styles should be handled in `src/scss/components/password-input/_index.scss`.  Please remove from this file.

Reply:
Changes have been done, please review.